PSKoans/Public/Measure-Karma.ps1

```diff
@@ -48,7 +48,10 @@
         }
         "OpenFolder" {
             Write-Verbose "Opening koans folder"
-            if (Get-Command -Name 'Code' -ErrorAction SilentlyContinue) {
+            if ($env:PSKoans_EditorPreference = 'code-insiders') {
+                Start-Process -FilePath 'code-insiders' -ArgumentList $env:PSKoans_Folder -NoNewWindow
```

Comment:
This needs to handle the case where the user sets the variable and then uninstalls VS Code insiders, or in edge cases where it might not have been added to the path like it ought to.